alephium/contracts/sequence/sequence.ral

```diff
@@ -0,0 +1,62 @@
+// sequence started from 0, set `next = 0` when init contract state
+Abstract Contract Sequence(
+    mut next: U256,
+    mut next1: U256, // TODO: better name
+    mut next2: U256,
+    undoneSequenceTemplateId: ByteVec,
+    refundAddress: Address
+) extends Constants() {
+    @using(readonly = true)
+    fn setExecuted(offset: U256, current: U256) -> U256 {
```

Comment:
[Nit] If this method is `setExecuted` perhaps `UndoneSequence` can be named `UnExecutedSequence`? Or perhaps `UnfinishedSequence`. 

Undone initially gives me the impression that it is gonna un-do an operation :)

Reply:
Sounds good to me 👍